app/models/station.rb

```diff
@@ -1,4 +1,3 @@
 class Station < ActiveRecord::Base
-  reverse_geocoded_by :latitude, :longitude, address: :location
-  after_validation :reverse_geocode
```

Comment:
The Geocoder module includes a method, `to_coordinates`, which takes an objects latlong attributes, and generates a geocoordinates array for each object, which other module methods use to make distance calculations, etc.